### PR TITLE
New gitbase-playground chart

### PR DESCRIPTION
**Pull request by rporres**
See `README.md` for some details on how the chart works.

### CODE REVIEW

**Comment by bzz** [comment]
@rporres `assigned carlosms and smacker an hour ago` - may be it's better to add ppl in reviewers? 

It will then show up on their https://github.com/pulls/review-requested and will rise the chances of beeing looked at.

**Comment by rporres** [comment]
A mistake, @bzz. Fixed.

Thx for the heads-up

File: gitbase-playground/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+              value: "{{ .Values.service.internalPort }}"
+            - name: GITBASEPG_ENV
+              value: "{{ .Values.settings.gitbasePlaygroundEnv }}"
+            - name: GITBASEPG_DB_CONNECTION
+            {{- if .Values.settings.gitbaseServer.password }}
+              value: {{ printf "%s:%s@tcp(%s:%s)/%s?maxAllowedPacket=%s" $user .Values.settings.gitbaseServer.password $address $port $database .Values.gitbasePlayground.gitbaseMaxAllowedPacket }}
+            {{- else }}
+              value: {{ printf "%s@tcp(%s:%s)/%s?maxAllowedPacket=%s" $user $address $port $database .Values.gitbasePlayground.gitbaseMaxAllowedPacket }}
+            {{- end }}
+            - name: GITBASEPG_BBLFSHD_URL
```
Comment (carlosms) [inline gitbase-playground/templates/deployment.yaml]: This should be [`GITBASEPG_BBLFSH_SERVER_URL`](https://github.com/src-d/gitbase-playground/blob/master/docs/CONTRIBUTING.md#configuration)

File: gitbase-playground/templates/statefulset.yaml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+          image: "{{ .Values.gitbasePlayground.image.repository }}:{{ required "Missing gitbasePlayground.image.tag" .Values.gitbasePlayground.image.tag }}"
+          imagePullPolicy: {{ .Values.gitbasePlayground.image.pullPolicy }}
+          env:
+            - name: PORT
+              value: "{{ .Values.service.internalPort }}"
+            - name: GITBASEPG_ENV
+              value: "{{ .Values.gitbasePlayground.env }}"
+            - name: GITBASEPG_DB_CONNECTION
+              value: "gitbase@tcp(127.0.0.1:3306)/none?maxAllowedPacket={{ .Values.gitbasePlayground.gitbaseMaxAllowedPacket }}"
+            - name: GITBASEPG_BBLFSHD_URL
```
Comment (carlosms) [inline gitbase-playground/templates/statefulset.yaml]: Same as before, it should be [`GITBASEPG_BBLFSH_SERVER_URL`](https://github.com/src-d/gitbase-playground/blob/master/docs/CONTRIBUTING.md#configuration)

File: gitbase-playground/templates/deployment.yaml (hunk 1 above)
Comment (rporres) [reply gitbase-playground/templates/deployment.yaml]: fixed in 8a1038b

File: gitbase-playground/templates/statefulset.yaml (hunk 2 above)
Comment (rporres) [reply gitbase-playground/templates/statefulset.yaml]: fixed in 8a1038b

**Comment by rporres** [comment]
fixed issues, @carlosms

**Comment by bzz** [comment]
Linking here just for convenience of navigation https://github.com/src-d/gitbase-playground/pull/82

**Pull request merged**